### PR TITLE
Tile load event

### LINKED ISSUE #5627: tileLoad event in Cesium3DTileset
We have `tileUnload` and `tileVisible`, a `tileLoad` will come in handy too.

### PR DESCRIPTION
**Pull request by lilleyse**
Fixes #5627

### CODE REVIEW

**Comment by pjcozzi** [comment]
@ggetz can you test and review this?

File: Source/Scene/Cesium3DTileset.js [hunk 1]
```diff
@@ -1223,7 +1239,10 @@ define([
 
         var removeFunction = removeFromProcessingQueue(tileset, tile);
         tile.contentReadyToProcessPromise.then(addToProcessingQueue(tileset, tile));
-        tile.contentReadyPromise.then(removeFunction).otherwise(removeFunction);
+        tile.contentReadyPromise.then(function() {
+            removeFunction();
+            tileset.tileLoad.raiseEvent(tile);
+        }).otherwise(removeFunction);
```
Comment (ggetz) [inline Source/Scene/Cesium3DTileset.js]: Since it looks like this can fail, would it be worth passing an error to the event?

**Comment by ggetz** [comment]
Other than that one comment, looks good.

File: Source/Scene/Cesium3DTileset.js (hunk 1 above)
Comment (lilleyse) [inline Source/Scene/Cesium3DTileset.js]: We would probably add a `tileFailed` event instead, a little more detail here: https://github.com/AnalyticalGraphicsInc/cesium/issues/5158

**Comment by lilleyse** [comment]
I want to keep this open for just a little longer, currently getting feedback on the forum.

File: Source/Scene/Cesium3DTileset.js (hunk 1 above)
Comment (ggetz) [inline Source/Scene/Cesium3DTileset.js]: OK, makes sense!

**Comment by lilleyse** [comment]
There hasn't been too much activity on the [forum post](https://groups.google.com/forum/#!searchin/cesium-dev/https$3A$2F$2Fgithub.com$2FAnalyticalGraphicsInc$2Fcesium$2Fpull$2F5628%7Csort:relevance/cesium-dev/MJ8UObzxlGk/1zRDK9_zBAAJ) so I think this should go in before the release and if needed tweaked later.

@pjcozzi can you merge?

**Pull request merged**